CMakeLists.txt

```diff
@@ -65,6 +62,34 @@ elseif(MSVC)
     endif()
 endif()
 
+if(UR_ENABLE_TRACING)
+    add_compile_definitions(UR_ENABLE_TRACING)
+
+    # fetch xpti proxy library for the tracing layer
+    FetchContentSparse_Declare(xpti https://github.com/intel/llvm.git "f25cc4006f4da3acd9f4050cd150065ab0857f4d" "xpti")
```

Comment:
Can you use the name of the tag here, like "sycl-nightly/20230223" instead? This would tell us more how old is the current version of XPTI for possible future updates. However, if we don't plan to update XPTI from time to time, then it's irrelevant.

Reply:
When I was writing this patch, I had to use this specific tag because it was the first commit with my fixes for the xptifw build system. Ideally, there'd be a tag for xpti version with a stable ABI, but no such luck.
I'll use a daily tag.